### PR TITLE
feat: added rating component

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
- created two different types of rating components
- added the numbered rating 
- added the star icon rating 
ratings-preview

### CODE REVIEW

**Review by rahulcs754 — APPROVED** [review]
I looked your good i think all look good in this code

File: components/rating.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@import url(/stylesheets/main.css);
+
+.ratings-container{
+    width: 30rem;
+    background-color: hsl(var(--primary-lighter));
```
Comment (coldpigli) [inline components/rating.css:5]: No need of defining hsl this way if you're not dealing with opacity.
Can simple say background-color: var(--primary-lighter);

File: components/rating.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    display: flex;
+    justify-content: center;
+    color: rgb(255,164,29);
+}
+
+.numbered-rating{
+    background-color: hsl(var(--green-color));
```
Comment (coldpigli) [inline components/rating.css:12]: Same as above

**Review by coldpigli — APPROVED** [review]
Suggested minor corrections

File: components/rating.css [hunk 3]
```diff
@@ -0,0 +1,14 @@
+@import url(/stylesheets/main.css);
+
+.ratings-container{
+    width: 30rem;
+    background-color: hsl(var(--primary-lighter));
+    display: flex;
+    justify-content: center;
+    color: rgb(255,164,29);
+}
```
Comment (Sameony) [inline components/rating.css:9]: If u are using the variables for colors then you might wanna change this to a variable too for the sake of consistency

**Review by Sameony — APPROVED** [review]
Love the clean structure. Do follow up on the naming and hsl issues.

File: components/rating.css (hunk 1 above)
Comment (vaibhav-mahalle) [reply components/rating.css:5]: if I declare the variables directly in hsl then I wont be able to use them while i need different opacity.

File: components/rating.css (hunk 2 above)
Comment (vaibhav-mahalle) [reply components/rating.css:12]: addressed your concerns in the above comments.

File: components/rating.css (hunk 3 above)
Comment (vaibhav-mahalle) [reply components/rating.css:9]: changed the color name to appropriate naming conventions in latest commits.

Comment (vaibhav-mahalle) [reply components/rating.css:9]: made necessary changes in latest commits.

**Pull request merged**